backend/src/helpers/config.js

```diff
@@ -0,0 +1,65 @@
+require('dotenv').config({ path: '../.env' });
```

Comment:
I'm not sure a config.js would be considered a helper file.
I have added this to our meeting to discuss.  Although i think i'm ok with this for now, i dont think we should leave it like this.